Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/_Template.cs

```diff
@@ -14,27 +14,47 @@ namespace Xamarin.Forms.Controls.Issues
 #endif
 	[Preserve(AllMembers = true)]
 	[Issue(IssueTracker.Github, 1, "Issue Description", PlatformAffected.Default)]
-	public class Issue1 : TestContentPage // or TestMasterDetailPage, etc ...
+	public class Github1 : TestContentPage // or TestMasterDetailPage, etc ...
```

Comment:
Historically, we've used `Issue` instead of `GitHub` for issues tracked in GitHub. (GitHub was the original issue tracker, before things were moved to Bugzilla.) For consistency, I'm inclined to say we should stick with `Issue` in the template. 

That said, I'm not strongly against the more logical `GitHub` prefix, but if we're going to use it, we should case it correctly (`GitHub` instead of `Github`).

Reply:
Okay, having talked with @samhouts on this, I think we should stick with `Issue` for GitHub issues. Otherwise, we'll end up with weird ordering in the IDEs for tests, which makes it inconvenient to find the ones you're looking for.

Reply:
I'll revert this back to `Issue` later today. Thanks.